UnityGLTF/Assets/UnityGLTF/Scripts/GLTFComponent.cs

```diff
@@ -122,6 +132,15 @@ public async Task Load()
 						renderer.sharedMaterial.shader = shaderOverride;
 					}
 				}
+
+				if (PlayAnimationOnLoad)
+				{
+					Animation[] animations = sceneImporter.LastLoadedScene.GetComponents<Animation>();
+					foreach (Animation anim in animations)
```

Comment:
I suggest having a property for choosing a specific animation index. Play all animations can be overwhelming

Reply:
There is no linkage here between animations and their indices. This is something we may want to add in the future